### PR TITLE
Add more prettier compatibility

### PR DESCRIPTION
**Pull request by jimmy-btn**
- [ ] Updated CHANGELOG.md
- [ ] Updated version in `package.json`
- [ ] Solemly swear to cut a release/tag after merge to master

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,7 +24,12 @@ module.exports = {
     "require-yield": "off",
 
     // Single quotes instead of double quotes
-    "quotes": ["error", "single", { "allowTemplateLiterals": true }],
+    "quotes": ["error", "single", {
+	"allowTemplateLiterals": true,
+
+	// Allow use of double quotes if it avoids escaping a single quote.
+	"avoidEscape": true
```
Comment (griffinmyers) [inline index.js]: my preference is to use backticks for strings that have single quotes.

File: index.js [hunk 2]
```diff
@@ -42,6 +47,12 @@ module.exports = {
     // Place spaces after keywords
     "keyword-spacing": "error",
 
+     // Allow mixing operators. Risky, but required for prettier.
+    "no-mixed-operators": "off",
```
Comment (griffinmyers) [inline index.js]: An alternative would be to use a `prettier-ignore` line in these places.  Not sure what the violations look like specifically, I'll leave it up to your judgement whether or not that makes sense.

File: index.js [hunk 3]
```diff
@@ -42,6 +47,12 @@ module.exports = {
...
+
+    // Allow multiple variables to be intialized on a single line.
+    "one-var-declaration-per-line": "off",
```
Comment (griffinmyers) [inline index.js]: My preference is for each declaration to have it's own standalone line:

```javascript
// : )
let a;
let b;

// : (
let a, b;

// : (
let a,
    b;
```

File: index.js (hunk 1 above)
Comment (jimmy-btn) [reply index.js]: That works for me. It makes converting over to prettier slightly more work, but I think I have some ideas there.

**Pull request closed without merging**